pkgs/applications/networking/cluster/terraform-providers/data.nix

```diff
@@ -600,13 +744,13 @@
       version = "1.5.0";
       sha256  = "0yhv9ahj6ajspgnl2f77gpyd6klq44dyl74lvl10bx6yy56abi2m";
     };
-  random =
```

Comment:
it looks like a bunch of providers are missing, mostly the old ones

Reply:
I invoked `hub api --paginate https://api.github.com/orgs/terraform-providers/repos | jq`, and couldn't find `random` etc in there. Is the GitHub API not showing some repos for some reason?